ddtrace/tracer/span.go

```diff
@@ -225,6 +225,44 @@ func (s *Span) SetTag(key string, value interface{}) {
 	s.setMeta(key, fmt.Sprint(value))
 }
 
+// Tag returns the value for a given tag key.
+// If the key does not exist, Tag returns nil.
+// All values may be of type string, float64, or int32 (only for ext.Error).
+// If the original tag value was not one of these types, Tag will return a string
+// representation of the value.
+// Use ext.Error to check if the span has an error, and ext.ErrorMsg, ext.ErrorType,
+// and ext.ErrorStack to get the error details.
+func (s *Span) Tag(k string) interface{} {
+	if s == nil {
+		return nil
+	}
+	s.RLock()
+	defer s.RUnlock()
+	switch k {
+	case ext.AnalyticsEvent, ext.EventSampleRate:
+		return s.metrics[ext.EventSampleRate]
+	case ext.SamplingPriority, ext.ManualDrop, ext.ManualKeep:
+		return s.metrics[keySamplingPriority]
```

Comment:
I think we could omit these 2 cases, since I don't think there will be many real use cases where this tag needs to be queried.

Also, the user probably wants to see the actual tags set in the span, (e.g. if `manual.drop` is actually not being set and instead it is `sampling.priority`), and that can be achieved with lines 260-262.

Reply:
Removed the two cases.